index.ts

```diff
@@ -56,7 +58,7 @@ const plugin: Plugin = {
                         event.timestamp && timestamp && new Date(event.timestamp) < new Date(timestamp)
 
                     // same ip as is currently set on the person
-                    const isSameIp = ip === event.ip
+                    const isSameIp = ip === ip
```

Comment:
question: Wouldn't this always resolve as true since the `ip` variable name is reused and overridden? 

It is now initially defined on line 10 and overridden on line 53.

Reply:
Good catch! Fixed in https://github.com/PostHog/posthog-plugin-geoip/pull/9